test/sql/updates/post.continuous_aggs.sql

```diff
@@ -3,7 +3,48 @@
 -- LICENSE-APACHE for a copy of the license.
 
 \x on
-SELECT * FROM mat_before ORDER BY bucket, location;
+SELECT
+    bucket,
+    location,
+    min_allnull,
+    max_temp,
+    agg_sum_expr,
+    avg_humidity,
+    stddev,
+    bit_and,
+    bit_or,
+    bool_and,
+    every,
+    bool_or,
+    count_rows,
+    count_temp,
+    count_zero,
+    corr,
+    covar_pop,
+    covar_samp,
+    regr_avgx,
+    regr_avgy,
+    regr_count,
+    regr_intercept,
+    regr_r2,
+    regr_slope,
+    regr_sxx,
+    regr_sxy,
+    round(regr_syy) AS regr_syy,
+    stddev_temp,
+    round(stddev_pop) AS stddev_pop,
+    stddev_samp,
+    round(variance) AS variance,
+    round(var_pop) AS var_pop,
+    round(var_samp) AS var_samp,
+    last_temp,
+    last_hl,
+    first_hl,
+    histogram
```

Comment:
Why do we need these? Are the tests flaky?

Reply:
Exactly

Reply:
You're correct... the better place to call round is in the CAgg definition. Fixed it!